enterprise/settings/test.py

```diff
@@ -162,6 +164,8 @@ def root(*args):
 
 MKTG_URLS = {}
 
+ENTERPRISE_CUSTOMER_CATALOG_DEFULT_CONTENT_FILTER = CUSTOMER_CATALOG_DEFULT_CONTENT_FILTER
```

Comment:
nit: use a dict directly instead of importing from enterprise constants as this is a settings file.